script/script.js

```diff
@@ -0,0 +1,17 @@
+ document.querySelector('#create_task').addEventListener('click', myClick);
+
+ function myClick(){
+    //console.log('work')
+    let text = document.querySelector('.i-1').value;
+    //console.log(a);
+    //document.querySelector('.out').innerHTML = ; тут іннер хтмл 
+    // він заміняє весь контент в пісі-какі-попі. мені треба 
+   
+
+   const piratik = document.createElement('li');
```

Comment:
It is important to give good variable names so that a new person (and you in 5 months) can easily understand your code.

piratik does not make any sense. Ask a question, what are you storing in this variable?
My guess you are storing: newTaskListItemElement

Reply:
Thenk you very much

Reply:
Thank you very much. I realized that writing variable names is illogical for my work.